docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -3,10 +3,16 @@
 
 [role="_abstract"]
 
-A devfile describes and defines a cloud native development environment.
+Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development while providing a way for being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
 
-Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple cloud native resources. Any number of identical workspaces can be created from the same devfile. They can be shared in various ways including a devfile registry or together with the source code of a project.
+Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, any number of identical workspaces can be created from the same devfile. This way, workspaces can be used by different teams and shared in various ways, including a devfile registry or together with the source code of a project.
 
+The benefits of devfiles include:
+
+* Reducing the gap between development and deployment
+* Finding stacks using the devfile registry
```

Comment:
I don't know if I would phrase `Finding stacks using the devfile registry` this way.

While I think a registry of stacks is certainly a benefit to using devfiles, I think it makes more sense to say something like "A registry for finding and hosting devfiles?"

Reply:
@johnmcollier, thanks. I like your rephrase, but for consistency, I want to have every bullet begin with a gerund. What do you think about:
`Having a registry fro finding and hosting devfiles`

Reply:
Yup, I think that  sounds good 👍